### PR TITLE
[MOSIP-25290] added new modules jobs to postgres-init charts

### PR DESCRIPTION
**Pull request by Rakshitha650**

### CODE REVIEW

File: charts/postgres-init/values.yaml [hunk 1]
```diff
@@ -241,3 +241,47 @@ databases:
...
+    port: 5432
+    su:
+      user: postgres
+      secret:
+        name: postgres-postgresql
+        key: postgresql-password
+    dml: 0
+
+  mosip_digitalcard:
+    enabled: true
```
Comment (ckm007) [inline charts/postgres-init/values.yaml:268]: Please make this false by default.

File: charts/postgres-init/values.yaml [hunk 2]
```diff
@@ -241,3 +241,47 @@ databases:
...
+    port: 5432
+    su:
+      user: postgres
+      secret:
+        name: postgres-postgresql
+        key: postgresql-password
+    dml: 0
+
+  mosip_idp:
+    enabled: true
```
Comment (ckm007) [inline charts/postgres-init/values.yaml:279]: please make this default as well.

File: charts/postgres-init/values.yaml [hunk 3]
```diff
@@ -241,3 +241,47 @@ databases:
...
+    port: 5432
+    su:
+      user: postgres
+      secret:
+        name: postgres-postgresql
+        key: postgresql-password
+    dml: 0
+
+  mosip_toolkit:
+    enabled: true
```
Comment (ckm007) [inline charts/postgres-init/values.yaml:257]: by default the same needs to be false

**Review by syedsalman3753 — APPROVED** [review]

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]
this PR is approved .. but will need the respective changes in 
1. digitalcard repo for running db scripts to create only digitalcard DB.
2. idp for unning db scripts to create only idp DB.
3. compliance for running db scripts to create only compliance DB.
The PR will be merged once the above mentioned changes are also added.

**Pull request closed without merging**